Xamarin.Forms.Core/ContentPage.cs

```diff
@@ -22,5 +22,19 @@ protected override void OnBindingContextChanged()
 				SetInheritedBindingContext(content, BindingContext);
 			}
 		}
+
+		internal override void OnControlTemplateChanged(ControlTemplate oldValue, ControlTemplate newValue)
+		{
+			if (oldValue == null)
+				return;
+
+			base.OnControlTemplateChanged(oldValue, newValue);
```

Comment:
Why not have this before line 28? Is it always the case that the base class will similarly do nothing if the `oldValue` is null?